lib/portlayer/storage/nfs/target.go

```diff
@@ -18,13 +18,19 @@ import (
 	"io"
 	"net/url"
 	"os"
+	"time"
 
 	nfsClient "github.com/vmware/go-nfs-client/nfs"
 	"github.com/vmware/go-nfs-client/nfs/rpc"
 
 	"github.com/vmware/vic/pkg/trace"
 )
 
+const (
+	// nfs client volume store mount idle timeout duration, this should protect us from waiting too long on idle NFS servers
+	nfsClientIdleTimeout = 2 * time.Minute
```

Comment:
I suspect that this is too high and we will have other timeouts that kick in before this lapses.
Really this needs to be configurable, so that it can be extended in very busy environments but is low for good responsiveness in most environments.